main.js

```diff
@@ -1,3 +1,17 @@
 import './style.css';
 
-console.log('Hello world');
+function tooltip_enable(event) {
+    var tooltip = document.createElement("span")
+    tooltip.innerHTML = "Hello World!";
+    tooltip.setAttribute("class", "tooltip");
```

Comment:
DOM api has special methods to work with classes.
You can add new class like so:
```javascript
tooltip.classList.add('tooltip')
```

More info here:
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList